app/templates/search/_categories.html

```diff
@@ -1,17 +1,17 @@
-{%- macro top_level_link(text, url) -%}
+{%- macro top_level_link(text, count, url) -%}
   <a href="{{ url }}" class="lot-filters__top-level-link">{{ text }}</a>
 {%- endmacro -%}
 
-{%- macro list_item__unselected(text, url) -%}
-  <li><a href="{{ url }}">{{ text }}</a></li>
+{%- macro list_item__unselected(text, count, url) -%}
+  <li>{% if count > 0 %}<a href="{{ url }}">{% endif %}{{ text }} ({{ count }}){% if count > 0 %}</a>{% endif %}</li>
```

Comment:
It'd be less verbose just to write `{% if count %}` but I guess this is personal preference :)